Sources/Workspace/UserToolchain.swift

```diff
@@ -40,6 +40,19 @@ public struct UserManifestResources: ManifestResourceProvider {
         self.libDir = libDir
         self.sdkRoot = sdkRoot
     }
+
+    /// Creates the set of manifest resources associated with a `swift` executable.
+    ///
+    /// - Parameters:
+    ///     - swiftExectuable: The absolute path of the associated `swift` executable.
+    public init(swiftExectuable: AbsolutePath) throws {
```

Comment:
Let's call this swiftCompiler for consistency with the other initializer.

Reply:
The other initializer is the path of `swiftc`. This one is the path of `swift`. (Though at the moment any file in the same directory happens to work.)

I’m not sure how best to make the difference clear.

Reply:
We should just use `swiftc` here.

Reply:
Why? `swift` is the one outsiders are familiar with.

Oh well, I guess any method of finding `swift` (`which`, etc.) *should* also be able to find `swiftc` 🤞. I’ll switch it and update the documentation to match.

Reply:
Switched it and pushed. (I hope I found all the places it used to say “swift”.)